src/app/api/senior/[id]/route.schema.ts

```diff
@@ -16,7 +16,8 @@ export const seniorDeleteResponse = z.discriminatedUnion("code", [
 ]);
 
 export const patchSeniorSchema = z.object({
-  name: z.string(),
+  firstname: z.string(),
```

Comment:
`patchSeniorSchema` and `postSeniorSchema` shares the same fields. Should we consider moving it to `model/`?